odoo/addons/base/models/ir_attachment.py

```diff
@@ -487,14 +487,15 @@ def _read(self, fields):
     def write(self, vals):
         self.check('write', values=vals)
         # remove computed field depending of datas
-        for field in ('file_size', 'checksum'):
+        for field in ('file_size', 'checksum', 'store_fname'):
             vals.pop(field, False)
         if 'mimetype' in vals or 'datas' in vals:
             vals = self._check_contents(vals)
         return super(IrAttachment, self).write(vals)
 
     def copy(self, default=None):
         self.check('write')
+        default = dict(default or {}, datas=self.datas)
```

Comment:
@mart-e This specific line breaks the image edition feature of the website builder in all versions (well at least starting from 14.0 where there are image edition features).

-> Add a snippet like text - image
-> Click on an image
-> Change the "filter"
-> Save -> Traceback : https://cdn.discordapp.com/attachments/689775172080762928/933765259594985542/unknown.png

No idea why, I don't understand your diff well enough :grimacing: Could you help? Thanks :+1:

Reply:
@qsm-odoo could you check with #83084 (and #83134 for upper) if it solves ?

Reply:
Seems solved indeed :+1: Thanks :+1: